test/AppLifecycle/ApiTests.cpp

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include "pch.h"
+
+using namespace winrt::Windows::ApplicationModel;
+using namespace winrt::Windows::ApplicationModel::Activation;
+using namespace winrt::Microsoft::ProjectReunion;
+using namespace Microsoft::VisualStudio::CppUnitTestFramework;
+
+namespace ProjectReunionCppTest
+{
+    TEST_CLASS(AppLifecycleApiTests)
+    {
+#if PRTEST_MODE_UWP
+        // UWP currently is not supported by these tests.
+        TEST_METHOD(GetActivatedEventArgsIsNull)
+        {
+            Assert::IsTrue(winrt::Microsoft::ProjectReunion::AppLifecycle::GetActivatedEventArgs() == nullptr);
+        }
+#else
+        TEST_METHOD(GetActivatedEventArgsIsNotNull)
+        {
+            Assert::IsTrue(winrt::Microsoft::ProjectReunion::AppLifecycle::GetActivatedEventArgs() != nullptr);
+        }
+
+        TEST_METHOD(GetActivatedEventArgsForLaunch)
+        {
+            auto args = winrt::Microsoft::ProjectReunion::AppLifecycle::GetActivatedEventArgs();
+            Assert::IsTrue(args != nullptr);
+            Assert::IsTrue(args.Kind() == ActivationKind::Launch);
+
+            auto launchArgs = args.as<LaunchActivatedEventArgs>();
```

Comment:
Is this a Microsoft type or a Windows type here? The usings above confuse me.

Reply:
The interfaces are defined by the platform.  The concrete types are implemented by this library.  So the interfaces are Windows types, and the concreate types are Microsoft.  Does that help clarify?